src/main/java/com/rakovets/course/java/core/practice/string/StringUtil.java

```diff
@@ -0,0 +1,205 @@
+package com.rakovets.course.java.core.practice.string;
+
+import java.util.Locale;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.Arrays;
+
+public class StringUtil {
+
+        // TASK_1: вывести строку, которая является результатом склеивания двух строк.
```

Comment:
You couldn't duplicate task description.
```suggestion
        // TASK_1
```